src/main/java/org/jenkinsci/plugins/github/webhook/subscriber/DefaultPushGHEventSubscriber.java

```diff
@@ -82,8 +87,17 @@ protected void onEvent(GHEvent event, String payload) {
             ACL.impersonate(ACL.SYSTEM, new Runnable() {
                 @Override
                 public void run() {
-                    for (AbstractProject<?, ?> job : Jenkins.getInstance().getAllItems(AbstractProject.class)) {
-                        GitHubTrigger trigger = job.getTrigger(GitHubPushTrigger.class);
+                    for (Job<?, ?> job : Jenkins.getInstance().getAllItems(Job.class)) {
```

Comment:
[This idiom](https://github.com/jenkinsci/jenkins/blob/252e1296099e838b8fba52221910442e0a757db4/core/src/main/java/jenkins/triggers/ReverseBuildTrigger.java#L261-263) is shorter, but I agree it would be nice to have a single `getTrigger(Class)` for convenience.